packages/ui/src/components/CookieBanner/CookieBanner.tsx

```diff
@@ -32,32 +31,25 @@ export const CookieBanner = (props: CookieBannerProps) => {
       bg="gray.700"
```

Comment:
Darker color would be better.

```suggestion
      bg="gray.900"
```